runtime/util_core/vmargs_core.c

```diff
@@ -277,8 +278,10 @@ addXOptionsFile(J9PortLibrary* portLib, const char *xOptionsfileArg, J9JavaVMArg
 		return -1;
 	}
 
-	/* add the -Xoptionsfile argument */
-	optArg = newJavaVMArgInfo(vmArgumentsList, NULL, ARG_MEMORY_ALLOCATION|CONSUMABLE_ARG);
+	/* add the -Xoptionsfile argument. Sets the USER_ARG flag if the optionfile is not the default optionfile */
+	optArg = newJavaVMArgInfo(vmArgumentsList, NULL, ARG_MEMORY_ALLOCATION | ((containsUserArgs ? USER_ARG : 0) | CONSUMABLE_ARG));
```

Comment:
nitpick: extra bracket not needed. i.e. `ARG_MEMORY_ALLOCATION | (containsUserArgs ? USER_ARG : 0) | CONSUMABLE_ARG`